.env

```diff
@@ -0,0 +1,13 @@
+REACT_APP_DOMAIN = dev-uzdmt05n.auth0.com
```

Comment:
@Greyflanel rename this file to .env.local for it to be caught by the .gitignore, otherwise it will be publically pushed to Github